app/src/main/java/org/jtaccuino/app/Studio.java

```diff
@@ -81,15 +81,38 @@ public Parent createParentNode() {
                 ToolBar.createMainToolBar()));
         bp.setBottom(StatusLine.getDefault().getNode());
 
-        sheetPane.getTabs().add(new TabSheet(SheetManager.getDefault().of(notebook)));
+        initTabs();
 
         return bp;
     }
 
-    public void shutdown() {
+    private void initTabs() {
+        var openFiles = FilePersistence.getDefault().openFiles;
+        if (openFiles.isEmpty()) {
+            sheetPane.getTabs().add(new TabSheet(SheetManager.getDefault().of(NotebookPersistence.INSTANCE.of())));
+        } else {
+            openFiles.forEach(file
+                    -> SheetManager.getDefault().open(NotebookPersistence.INSTANCE.of(file.path().toFile()))
```

Comment:
What happens here if one of the files in the openFiles list no longer exists?
Exception and exit?

Reply:
Do not open file. Looks like we need notifications...

Reply:
Not necessarily.
NetBeans IDE for example simply doesn't open no longer existing files.
VS Code however does open an editor for a missing file but marks it as deleted by strikethrough formatting of the editor name. It however can reactivate the editor once the file is reinstated on disk.

Reply:
Ok. Check if exists else just skip.